src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -86,4 +87,35 @@ private static void setCheckName(InputConfiguration.Builder inputConfigBuilder,
         final String checkPath = getPackageFromFilePath(filePath) + "." + checkName + "Check";
         inputConfigBuilder.setCheckName(checkPath);
     }
+
+    private static void setCheckProperties(InputConfiguration.Builder inputConfigBuilder,
+                                      String filePath, List<String> lines)
+                    throws CheckstyleException {
+        int lineNo = 2;
+        for (String line = lines.get(lineNo); !line.contains("*/"); line = lines.get(++lineNo)) {
+            if (line.isEmpty()) {
+                continue;
+            }
+            if (line.contains("(default)")) {
```

Comment:
isn't it redundant? you have inner condition `if (line.contains("(default)")) {`

Reply:
I'm so sorry. Missed that while refactoring. Fixed now.